test/python/WMCore_t/Services_t/MicroService_t/Unified_t/Transferor_t.py

```diff
@@ -14,22 +14,25 @@
 from WMCore.Services.PhEDEx.PhEDEx import PhEDEx
 from WMCore.Services.MicroService.Unified.Transferor import \
         RequestStore, RequestManager
-
+from WMQuality.Emulators.PhEDExClient.MockPhEDExApi import MockPhEDExApi
 
 class TransferorTest(unittest.TestCase):
```

Comment:
in order to use the mocked phedex, you need to use the `EmulatedUnitTestCase` super class instead.
Then you call the setUp method from the super class too, just as Seangchan suggested in the GH issue.

Reply:
and can you explain why do we need EmulatedUnitTestCase in order to use mocked phedex. I tried it and it works w/o EmulatedUnitTestCase and it works with EmulatedUnitTestCase. So my question is what's the point of using it?

Reply:
It worked without it either because a) it is still making real calls to the data service or b) because you're explicitly passing a mockphedex object to the src class code.

With EmulatedUnitTestCase, it uses the mock.patch to remap calls to PhEDEx to a MockPhedex instead, such that we don't need to have a special handle for tests in the src/ code